internal/worker/modelworkermanager/modelworkermanager.go

```diff
@@ -203,7 +198,12 @@ func (m *modelWorkerManager) Wait() error {
 }
 
 func (m *modelWorkerManager) loop() error {
-	watcher := m.config.ModelWatcher.WatchModels()
+	ctx := context.Background()
```

Comment:
Use a scoped context. See other workers for how `scopedContext()` is done.

Reply:
Sure thanks Ian :)